.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.5"
+  - "3.6"
+  - "3.7-dev"
+install:
+  - pip install -r tests-require.txt
+compiler:
+  - gcc
```

Comment:
I never called C++ inside Python. But since you are writing C++ code, does it make more sense to use `g++` here?